vortex-array/src/arrow/array.rs

```diff
@@ -177,6 +178,19 @@ impl FromArrowArray<&ArrowStructArray> for ArrayData {
     }
 }
 
+impl<O: OffsetSizeTrait + NativePType> FromArrowArray<&GenericListArray<O>> for ArrayData {
+    fn from_arrow(value: &GenericListArray<O>, nullable: bool) -> Self {
+        ListArray::try_new(
+            Self::from_arrow(value.values().clone(), value.values().is_nullable()),
+            // offsets are always non-nullable
```

Comment:
Still use the `from_arrow` constructor though. We should remove the other one tbh

Reply:
?

Reply:
`ArrayData::from_arrow(value.offsets().clone(), false)`